README.md

```diff
@@ -23,7 +23,7 @@ docker run -it
 	-v "/source-folder/":/source
 	-v "/destination-folder/":/destination
 	docker-organize
-	"* * * * * organize run"
+	"* * * * *"
```

Comment:
This will be deprecated in a future release when I get Docker environment variables working.